templates/index.tpl

```diff
@@ -24,13 +24,15 @@
                         <label for="audio"><span class="ui"></span>
                             {t}Audio only (MP3){/t}
                         </label>
+                       {if $config->convertSeek}
```

Comment:
If you only check that here, users are still able to send an HTTP request with these parameters.

You probably need to add a check here as well: https://github.com/Rudloff/alltube/blob/b02e255cb419cf6afbe89e8a82cf4713f06a3aae/classes/Controller/DownloadController.php#L104

Reply:
True.
Cant really test this as for some reason composer test always timeouts for me (tried some env withouts success): ```The process "phpunit" exceeded the timeout of 300 seconds.```

Maybe something like this might work?
```

        $from = null;
        $to = null;
        if ($this->config->convertSeek) {
            $from = $request->getQueryParam('from');
            $to = $request->getQueryParam('to');
        }
```

Reply:
>  Cant really test this as for some reason composer test always timeouts for me (tried some env withouts success): The process "phpunit" exceeded the timeout of 300 seconds.

Fixed in 4bd121cda2f05cc4efdc0958a9c936a24e8406cb.

> Maybe something like this might work?

Yes, this should work.

Reply:
added